internal/api/scim/resources/user_mapping.go

```diff
@@ -136,6 +136,15 @@ func (h *UsersHandler) mapPrimaryEmail(scimUser *ScimUser) *command.Email {
 		}
 	}
 
+	// if no primary email was found, the first email will be used
+	for _, email := range scimUser.Emails {
```

Comment:
How do we handle it if besides primary, there is no email provided at all?

Reply:
In Scim, we just set the email to nil. The command layer will then throw an error due to the missing email, which is required in Zitadel.

Reply:
True